build.gradle

```diff
@@ -1674,7 +1730,6 @@ project(':clients') {
 
   configurations {
     generator
-    shadowed
```

Comment:
I don't really know what this does. Can you explain?

Reply:
This is because moving shadowJar into the subprojects block required moving the dependencies forward, which is no longer needed in the new PR.